src/javafundamentals/lambdas/Ex07CalculatorWithLambdas.java

```diff
@@ -8,4 +10,17 @@
  */
 
 public class Ex07CalculatorWithLambdas {
+    public static void main(String[] args) {
+        Calculator myCalculator = new Calculator();
+        System.out.println(myCalculator.calculator((a, b)->(a*b), 4, 5));
```

Comment:
The exercise said to create variables so that you can reuse, you used the lambda in place instead of putting it into a variable, but ok ... hehhe
An example would be :
`BiFunction<Integer, Integer, Integer> biFunction multiplier = (a, b) -> (a * b)`